### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by MarlenAw**

### CODE REVIEW

**Comment by MynahMarie** [comment]
@MarlenAw Hey Marlen, good job on the styling of the home page! Just a few comment...

It looks like alot of comments but really its just color issues, once u decide on a color scheme it should be really easy to just apply that everywhere. ❤️

**Review by MynahMarie — CHANGES_REQUESTED** [review]
- The "View Reports" button: it's not clear that this is a button and it's clickable... maybe make it more like a button to encourage users to click on it.
- The text on the other pages dispappears because all the text is white
- I really like the background image for the home page but not sure for the other pages if we should keep it...
- the categories buttons are also on a white background, maybe you can decide on a different color?
- also the "Back" and "Next" buttons are on white background so whatever color u decide for the icons we could maybe use the same one for those buttons as well?
- finally, if you change the orange on the landing then we need to inform tachyons to always use that orange (it's still set as the previous orange on the style.css file

**Comment by MarlenAw** [comment]
@MynahMarie I'm still working on it! That's why I didn't assign you guys to review just yet

**Review by hoslack — APPROVED** [review]

**Review by MynahMarie — CHANGES_REQUESTED** [review]
Great job for the styling! The only thing I would adjust before merging is to change the color of the text on the thank you page! After that I think it's good to go :)

**Review by MynahMarie — APPROVED** [review]

**Comment by Karyum** [comment]
@MarlenAw can i review this right now ?

File: config/dev.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  mongoURL: ''
```
Comment (Karyum) [inline config/dev.js:2]: what is  this doing here?

**Review by Karyum — CHANGES_REQUESTED** [review]
the css is using loads of `px` that's not responsive, it might be for your browsers but not really for other. and are you 100% sure you don't want to do it with tachyons? to keep it consistent and and responsive (since tachyons adds the webkits and stuff )

**Comment by MarlenAw** [comment]
@Karyum I had to over write tachyons because it wasn't doing what I wanted

**Review by Karyum — APPROVED** [review]

**Comment by Karyum** [comment]
# Amazing job

**Pull request merged**